doc/components.md

```diff
@@ -259,9 +263,3 @@ The corners are drawn at the same size, the sides are stretched on the side dire
 Using this, you can get a box/rectangle that expands well to any sizes. This is useful for making panels, dialogs, borders.
 
 Check the example app `nine_tile_box` details on how to use it.
-
-## Effects
```

Comment:
Not sure about removing that, seems useful enough to keep.

Reply:
Re-added, thought it could be removed since we have a whole article about effects, but maybe it is good to have it linked here.